plugin/src/main/java/com/denizenscript/denizen/objects/properties/PropertyRegistry.java

```diff
@@ -146,6 +146,8 @@ public static void registermainProperties() {
         // register core MaterialTag properties
         if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_13)) {
             PropertyParser.registerProperty(MaterialAge.class, MaterialTag.class);
+            PropertyParser.registerProperty(MaterialPickle.class, MaterialTag.class);
+            PropertyParser.registerProperty(MaterialRepeater.class, MaterialTag.class);

```

Comment:
^ Started a second PR before finishing the first and didn't hide all the evidence, eh?

Reply:
OOF, lol